docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -0,0 +1,115 @@
+.. _installing_the_control_node:
+.. _from_yum:
+
+Installing Ansible on specific operating systems
+================================================
+
+Follow these instructions to install the Ansible community package on a variety of operating systems.
+
+Installing Ansible on CentOS or Fedora
+----------------------------------------------
+
+On Fedora:
+
+.. code-block:: bash
+
+    $ sudo dnf install ansible
+
+On CentOS:
+
+.. code-block:: bash
+
+    $ sudo yum install epel-release
+    $ sudo yum install ansible
+
+RPMs for currently supported versions of CentOS are also available from `EPEL <https://fedoraproject.org/wiki/EPEL>`_.
+
+Ansible can manage older operating systems that contain Python 2.6 or higher.
```

Comment:
Is this comment specific to CentOS? I'd say "older CentOS versions" if so, or move it if not.